tests/test_utils.py

```diff
@@ -408,7 +409,89 @@ def test_lcc_bad_matrix(self):
 
 
 def test_binarize():
```

Comment:
I don't think this test is getting run right now? `pytest` wasn't imported before.